slack/src/helpers/ScheduleReminder.ts

```diff
@@ -1,19 +1,14 @@
 import axios, { AxiosResponse } from 'axios'
 import qs from 'querystring'
 
-export const DEFAULT_CRON_SCHEDULE: string = '0 8-22 * * *'
-
 export default class ScheduleReminder {
   static sendReminder (): Promise<AxiosResponse<any>> | void {
```

Comment:
This function never returns `void` now

```suggestion
  static sendReminder (): Promise<AxiosResponse<any>> {
```